src/renderer/utils/index.ts

```diff
@@ -2,6 +2,8 @@
 
 export const isElectron = !!navigator.userAgent.match(/Electron/);
 
+export type Disposer = () => void;
```

Comment:
Maybe better to create `utils/common-types.ts`?

Reply:
sure, good idea